### PR TITLE
docs: add buildx du verbose example

### PR DESCRIPTION
**Pull request by dvdksn**
Signed-off-by: David Karlsson <35727626+dvdksn@users.noreply.github.com>

### CODE REVIEW

File: docs/reference/buildx_du.md [hunk 1]
```diff
@@ -60,6 +60,30 @@ The asterisks (\*) in the output indicate the following:
...
+Mutable:  true
+Reclaimable:  true
+Shared:    false
+Size:    36.86kB
+Description:  mount / from exec /bin/sh -c htmltest
+Usage count:  1
+Last used:  22 hours ago
+Type:    regular
+
+...
```
Comment (crazy-max) [inline docs/reference/buildx_du.md:84]: Output doesn't seem aligned correctly:

```
...
Last used:      2 days ago
Type:           regular

ID:             05d0elirb4mmvpmnzbrp3ssrg
Parent:         e8sfdn4mygrg7msi9ak1dy6op
Created at:     2023-11-20 09:53:30.881558721 +0000 UTC
Mutable:        false
Reclaimable:    true
Shared:         false
Size:           0B
Description:    [gobase 3/3] WORKDIR /src
Usage count:    3
Last used:      24 hours ago
Type:           regular

Reclaimable:    4.453GB
Total:          4.453GB
```

Comment (dvdksn) [reply docs/reference/buildx_du.md:84]: yea it's weird I got this misalignment in my terminal as well 🤔

Comment (dvdksn) [reply docs/reference/buildx_du.md]: updated

**Review by crazy-max — CHANGES_REQUESTED** [review]
Needs a `make docs` to align TOC with new section

**Review by crazy-max — APPROVED** [review]

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**